modules/Microsoft.Network/trafficmanagerprofiles/.test/common/deploy.test.bicep

```
@@ -0,0 +1,74 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes')
```

Also something to update in the utility
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes.')
```